skt/reporter.py

```diff
@@ -329,6 +328,15 @@ def __getjobresults(self):
                         'name': task_node.attrib.get('name'),
                         'logs': logs
                     }
+                    # Add 'boot test' to the kpkginstall task if it failed to
+                    # show that it includes booting the kernel, not just
+                    # installing it. We can't do it earlier because we need the
+                    # original task to work with.
+                    kpkginstall_name = '/distribution/kpkginstall'
+                    if failed_task_detail['name'] == kpkginstall_name:
+                        failed_task_detail['name'] = '{} (boot test)'.format(
```

Comment:
I think we should move this logic to the jinja template and do the renaming of the task just once. Here we are renaming it twice, on the success case and here on the failed case.